.github/workflows/ci.yml

```
@@ -6,9 +6,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Set up Go
-        uses: actions/setup-go@v2
+        uses: actions/setup-go@v3
         with:
-          go-version: ^1.16
+          go-version: 1.18
```

This v3 feature is handy, if appropriate: :shrug: 
```suggestion
          go-version-file: 'go.mod'
```

---

Oooo nice! Thanks!